docs/approach.rst

```diff
@@ -128,13 +133,15 @@ For the example voxel, the resulting weights are:
   :width: 400 px
   :align: center
 
+THese normalized weights are then used to compute a weighted average that takes advantage
+of the higher signal in earlier echoes and the heigher sensitivty at later echoes.
 The distribution of values for the optimally combined data lands somewhere
 between the distributions for other echoes.
 
 .. image:: /_static/a09_optimal_combination_value_distributions.png
 
 The time series for the optimally combined data also looks like a combination
-of the other echoes (which it is).
+of the other echoes (which it is). This optimally combined data is written out as **ts_OC.nii.gz**
```

Comment:
Can we do new sentences on new lines, please ?

Reply:
what the BDFL wants, the BDFL gets.